INSTALL.md

```diff
@@ -9,23 +9,31 @@ To install subscription-manager, run
 sudo dnf install subscription-manager
 ```
 
+This is not necessary on RHEL installations as subscription-manager is pre-installed and registration is required to install any packages with DNF.
+
 ## Developer installation
 
-The process below has been tested on Fedora 36.
-Other Fedora versions or distributions may require some adaptation.
+The process below has been tested on Fedora 36 and RHEL 9.
+Other Fedora/RHEL versions or distributions may require some adaptation.
```

Comment:
Let's just drop the `Fedora/RHEL`, at this point it's clear.

Reply:
Dropped.